### DISCUSSION TITLE
Make whipper less hard work?

### DISCUSSION THREAD

**Discussion by martinwguy** [whipper-team/whipper#607] [Ideas]
Hi! I'm comparing cd-rippers to conserve a 400-500 CD collection from 2015 found in an abandoned house, and am finding whipper to be -er- clunky to use. I'd rather just say "whipper cd rip" and have it work. Instead I have to say "whipper cd rip --offset 0 -U -k --cdr" (I finally got to the right collection of options!). I have to give it an invented offset because I tried "whipper offset find" with 4 different CDs, including "Lady Gaga's Gratest Hits" before I gave up and it always said "404 Not found in database". Hence -U

So my suggestion is to make whipper "just work" more, doing away with the --cdr and -U options (or making them default to true). As for the offset, i assume whipper can't find it out wothout the Accurip databae info, so maybe I have to try a few more CDs until I find one that it recognizes, but after trying 4 major CDs, my hopes are slim and it looks like a defect, so a default offset of 0 would be nicer, maybe printing a warning instead of failing.

Thanks & keep up the good work!

    M

**Comment by whiteawake** [comment]
A late response, but for anyone else this applies to: you can simply set an alias for `whipper` with any custom flags in your `.bashrc` file (or shell equivalent).

So for @martinwguy’s input, it would look something like this: 

`alias whipper='whipper cd rip --offset 0 -U -k --cdr'`

– and then all you would need to type is `whipper` in the terminal.

Although it’s safer and considered better practice to set the alias to a custom, unique name.

**Comment by martinwguy** [comment]
Thanks. I'd forgotten about whipper but will be happy to give it another try

as cdparanoia has the worrying habit of giving different results on 
every run.


Blessings


     M


BlMessage ID: 
***@***.***>
